### ISSUE TITLE
AttributeError: 'ZMQSocketChannel' object has no attribute 'execute'

### ISSUE THREAD

**Issue by jldork** [iiSeymour/Flasked-Notebooks#2]
Hey this is great stuff - I'm really excited by this concept.

I just tried running your example (made no code changes) and somehow got this error ZMQSocketChannel object has no attribute 'execute'. 

Is this something you have run into before? Could it be my IPython version?

Thanks!

**Comment by iiSeymour** [comment]
> Could it be my IPython version?

Yes, it looks that way. This project was written before the 3.x release which causes this exception.

**Comment by gpamfilis** [comment]
Has anyone found a solution to this?

**Comment by QuadTriangle** [comment]
`pip install --user ipython==2.4.1`
